### PR TITLE
Feature/SC-5463 Updating pagination logic

### PR DESCRIPTION
**Pull request by murat-merdoglu-hpi**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.schul-cloud.org/browse/SC-5463

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5804387&bid=18999617)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5804387&bid=18999617)

File: controllers/administration.js [hunk 1]
```diff
@@ -2778,17 +2778,24 @@ const getTeamMembersButton = (counter) => `
...
 
 	const itemsPerPage = parseInt(req.query.limit, 10) || 25;
 	const filterQuery = {};
 	const currentPage = parseInt(req.query.p, 10) || 1;
 
+	const dataLength = await api(req)
+		.get('/teams/manage/admin')
+		.then((dataResponse) => dataResponse.total );
+
+	const exceedDataLimit = ((itemsPerPage * (currentPage - 1)) > dataLength) ? true : false;
```
Comment (qnowak) [inline controllers/administration.js:2792]: Do you need that ? true : false; ?
```suggestion
	const exceedDataLimit = ((itemsPerPage * (currentPage - 1)) > dataLength);
```
It should also work the same way.

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5804387&bid=18999617)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5804387&bid=18999617)

**Pull request merged**